src/applications/Applications.DicomLoader/Applications.DicomLoader.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Most of the properties here will be inherited from `/Directory.Build.props` and `/src/Directory.Build.props`